scripts/generate_sudo_list.py

```diff
@@ -0,0 +1,64 @@
+"""Helper script to generate sudo list doc."""
+import filecmp
+import sys
+import tempfile
+from pathlib import Path
+
+import yaml
+
+
+def collect_sudo_commands():
+    """Collect all ansible cmds that require sudo."""
+    network_runner_path = Path("quipucords/scanner/network/runner")
+    list_cmds = []
+    for playbook in network_runner_path.glob("roles/*/tasks/main.yml"):
+        for task in yaml.safe_load(playbook.open()):
+            if "user_has_sudo" in task.get("when", ""):
```

Comment:
we are only collecting raw commands that require sudo.

I'd check for this in a slightly different way.
Instead of checking for our "user_has_sudo" I'd check for "become" == yes/true as this is the actual way to have a role run with a privilege escalation.